src/backend/renderer.js

```diff
@@ -1729,13 +1764,21 @@ export function attach(
     }
   }
 
+  function findCurrentFiberUsingSlowPathById(id: number): Fiber | null {
+    const fiber = idToFiberMap.get(id);
+    if (fiber) {
+      return findCurrentFiberUsingSlowPath(fiber);
+    }
+    return null;
+  }
```

Comment:
Looks like there aren't really any uses of `findCurrentFiberUsingSlowPath()` anymore. Let's just change that method to accept an `id` instead and we can get rid of this indirection here (and in `inspectElementRaw()`)

Reply:
I'd be down if we also rename the original `findCurrentFiberUsingSlowPath` to `...ById` then to highlight that it's different from what is copied from React.

Reply:
Yeah, the name change would seem appropriate to match our existing conventions too.